### PR TITLE
feat: setup CI for the PHP SDK

### PR DESCRIPTION
**Pull request by ayewo**
This PR fixes[^1] https://github.com/highlight/highlight/issues/7691

/claim https://github.com/highlight/highlight/issues/7691

[^1]: playing with the edit button because the Algora bot didn’t pick up that I submitted a PR that closed the issue.

### CODE REVIEW

**Comment by ayewo** [comment]
@Vadman97 please take a look when you have a moment.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: "Continuous Integration"
+
+on:
+  - push
+  - pull_request
```
Comment (Vadman97) [inline .github/workflows/ci.yml]:
```suggestion
on:
    push:
        branches: ['main']
    pull_request:
        types: [opened, synchronize]
```

**Review by Vadman97 — APPROVED** [review]
nice! could we ge the PR to run the checks to make sure they are good to go?

**Comment by Vadman97** [comment]
@ayewo is it possible to run this check in your fork to validate that it's working?

**Comment by ayewo** [comment]
Thanks for your review!

> nice! could we ge the PR to run the checks to make sure they are good to go?

If by "run the checks" you are referring to a way to trigger GitHub Actions on the `.github/workflows/ci.yml` file, it wont work *yet* until you merge the code :)

If OTOH you are referring to the tests, this is already the case in `.github/workflows/ci.yml` where line 31 executes `composer test` which calls the tests.

**Comment by ayewo** [comment]
> @ayewo is it possible to run this check in your fork to validate that it's working?

@Vadman97 
Here's a link to the run I triggered just now in my fork:
https://github.com/ayewo/highlight-php/actions/runs/7951988402/job/21837310205

**Pull request merged**